modules/post/windows/gather/enum_chocolatey_applications.rb

```diff
@@ -0,0 +1,94 @@
+# This module requires Metasploit: https://metasploit.com/download
```

Comment:
Looks like the linter detected this module is accidentally marked as executable:

> modules/post/windows/gather/enum_chocolatey_applications.rb - [ERROR] Module should not be executable (+x)

Reply:
I just fixed this. Not sure why it was executable, but it shouldnt be anymore